api/libproject/devstart.rb

```diff
@@ -64,6 +72,16 @@ def dev_up(args)
   common.run_inline_swallowing_interrupt %W{docker-compose up api}
 end
 
+def run_tests(args)
+  common = Common.new
+  common.docker.requires_docker
+
+  ensure_git_hooks
+  ensure_personal_gradle
+
+  common.run_inline %W{docker-compose run --rm api ./gradlew test} + args
```

Comment:
If I just want to run ./gradlew test directly instead, is that going to be a problem going forward?

Reply:
Shouldn't. These scripts are mainly to document things and make common tasks easy. They should never prevent you from other things you want to do.